### ISSUE TITLE
npm install -g angular-cli@webpack return 1.0.0-beta.14 instead of 1.0.0-beta.11-webpack.9-4

### ISSUE THREAD

**Issue by Chabane** [angular/angular-cli#2124]
Hi,

If I do npm install -g angular-cli@webpack, the version with broccoli is returned.

**Comment by ciesielskico** [comment]
Are you sure it's NOT the webpack version? There was a commit a couple of hours ago updating the version from 11.webpack.9-2 to version 14.

https://github.com/angular/angular-cli/commit/5a40a859c3ef2a3865a5f8bdf44b75204a3e110a

**Comment by ifaouibadi** [comment]
rm -rf node_modules dist tmp typings
npm install --save-dev angular-cli@webpack

then ``` ng init ``` to update your files
https://github.com/angular/angular-cli#working-with-master

**Comment by AloeDream** [comment]
Looks like beta-14 is a Webpack version now.

**Comment by ciesielskico** [comment]
Can confirm, just updated to beta.14, it is the webpack version.

**Comment by Chabane** [comment]
I got this error after running ng build -prod: 

```
 No angular-cli-build.js found. Please see the transition guide: https://github.com/angular-cli/angular-cli/blob/master/TRANSITION.md#user-content-brocfile-transition.
15-Sep-2016 12:10:51	[INFO] Error: No angular-cli-build.js found. Please see the transition guide: https://github.com/angular-cli/angular-cli/blob/master/TRANSITION.md#user-content-brocfile-transition.
```
With "angular-cli": "^1.0.0-beta.11-webpack.9-4",

It's fixed with : "angular-cli": "1.0.0-beta.14",
